demos/entry/vexflow.html

```diff
@@ -39,7 +39,7 @@
   .addClef('treble')
   .addTimeSignature('4/4');
 
-vf.draw();`;
+factory.draw();`;
```

Comment:
In general I approve of writing out `factory` instead of f or vf or vff.

However I'm not a huge fan of seeing `new VexFlow.ClassName()` everywhere. It can get wordy especially if we use multiple VexFlow classes. 

I think that's why we "extracted" the class names at the top of each example. The destructure assignment acts like an import at the top, so we don't have to have VexFlow.* repeated throughout the code.

Reply:
I don't think I ever added `Vexflow.` anywhere that there was more than one classname extracted.  I specifically left alone `const { Dot, Factory } = VexFlow` and `const { Factory, StaveHairpin } = VexFlow`.   Unless I made a mistake, I thought I only used `new Vexflow.Factory()` when Factory was the only attribute of VexFlow that was referenced.